notebooks/lesson03_data-cleaning.ipynb

```diff
@@ -0,0 +1,1866 @@
+{
```

Comment:
<p>the 20,640 seems to be misaligned with the 19448 in the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>good catch! a bug from a previous iteration</p>